packages/swingset-runner/src/main.js

```diff
@@ -156,11 +156,7 @@ function generateSwingsetBenchmarkConfig(
       benchmarkBootstrap: {
         sourceSpec: new URL('vat-benchmarkBootstrap.js', import.meta.url)
           .pathname,
-        parameters: {
-          config: {
-            bootstrap: baseConfig.bootstrap,
-          },
-        },
+        parameters: { config: { bootstrap: baseConfig.bootstrap } },
```

Comment:
```suggestion
        parameters: { config: { bootstrap: baseBootstrap } },
```
(and likewise below; `baseConfig.bootstrap` can/should be replaced with `baseBootstrap`)